egl/egl_util.c

```diff
@@ -30,6 +30,26 @@
 #include "libdrm/drm_fourcc.h"
 #endif
 
+static PFNEGLCREATEIMAGEKHRPROC createImageProc = NULL;
+static PFNEGLDESTROYIMAGEKHRPROC destroyImageProc = NULL;
+
+EGLImageKHR createImage(EGLDisplay dpy, EGLContext ctx, EGLenum target,
+                        EGLClientBuffer buffer, const EGLint *attrib_list)
+{
+    if (!createImageProc) {
```

Comment:
glEGLImageTargetTexture2DOES is part of the GL/ES/ES2 extensions, whereas eglCreateImageKHR and eglDestroyImageKHR are part of the EGL extensions.  Initializing all of them in one place would impose a dependency on GLES2 in egl_vaapi_image.cpp.